### PR TITLE
Update slogan & logo

### PR DESCRIPTION
**Pull request by 6543**
- Update woodpecker slogan to repo too
  we are not just a simple fork anymore 🎉
- use https://github.com/woodpecker-ci/design/blob/main/logo/non-square.svg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,13 +46,13 @@
 </p>
 <br/>
 
-Woodpecker is a community fork of the Drone CI system.
+Woodpecker is a simple CI engine with great extensibility.
```
Comment (anbraten) [inline README.md:49]: We could maybe even leave out the simple as users could think its not powerful.

Comment (6543) [reply README.md:49]: well I want to have the simple in it - as we try to be simple - without cuting down on being powerfull

Comment (6543) [reply README.md:49]: what about "... simple jet powerful ..." ?

Comment (6543) [inline README.md:49]:
```suggestion
Woodpecker is a simple jet powerful CI/CD engine with great extensibility.
```
?

Comment (pat-s) [reply README.md:49]: I get your idea with "simple" though it's quite a subjective word and people might understand it negatively (which is what @anbraten might have referred to).
(Same goes for "powerful" in the end.)

Maybe "golang-based" or "efficient" might also transmit the message? In the end people care about

- speed
- resource-efficiency
- extensibility/flexibility

and having these somehow reflected in this sentence would be great 🙂

Comment (anbraten) [reply README.md:49]: In terms of UX it could also be intuitive?

Comment (xoxys) [reply README.md:49]: It's just a single sentence... I would go with "Woodpecker is a simple yet powerful CI/CD engine with great extensibility." as a good compromise. IMO we tend to overcomplicate simple problems sometimes :see_no_evil:

**Comment by xoxys** [comment]
If we change the slogan in the readme to something else, this change should be reflected in all other places as well, see https://github.com/search?q=repo%3Awoodpecker-ci%2Fwoodpecker+%22Woodpecker+is+a+simple+CI+engine+with+great+extensibility%22&type=code

**Review by lafriks — APPROVED** [review]

**Review by xoxys — APPROVED** [review]

**Pull request merged**